app/Http/Controllers/TaskController.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\TaskStoreRequest;
+use App\Models\Task;
+use App\Models\Project;
+use Auth;
+
+class TaskController extends Controller
+{
+
+    public $projects;
+
+    public function __construct()
+    {
+        $this->projects = Project::orderBy('title')->get()->pluck('title', 'id');
```

Comment:
@faxunil you can use pluck without `get()`, please search my channel for video about pluck.